lib/Doctrine/Migrations/EventDispatcher.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations;
+
+use Doctrine\Common\EventArgs;
+use Doctrine\DBAL\Connection;
+use Doctrine\Migrations\Configuration\Configuration;
+use Doctrine\Migrations\Event\MigrationsEventArgs;
+use Doctrine\Migrations\Event\MigrationsVersionEventArgs;
+
+final class EventDispatcher
```

Comment:
I am personally not a big fan of introducing _yet another event dispatcher_ in Doctrine, given that we are looking for replacement of the one in Common for ORM 3.0.

I suppose this would be good as temporary solution until the replacement is found, but then it should be abstracted through interface + marked as interal.

Reply:
@Majkl578 this is not necessarily a new "EventDispatcher", it is a composition class that depends on the existing `Doctrine\Common\EventManager`. I am fine with marking it internal. I don't intend for anyone else to be using this outside of migrations internals.

Reply:
This is really just an abstraction on top of EventManager and not necessarily a new EventDispatcher. I will mark it as internal.